src/unix/notbsd/linux/musl/mod.rs

```diff
@@ -351,10 +308,18 @@ pub const SIG_UNBLOCK: ::c_int = 0x01;
 pub const FALLOC_FL_KEEP_SIZE: ::c_int = 0x01;
 pub const FALLOC_FL_PUNCH_HOLE: ::c_int = 0x02;
 
-pub const _SC_2_C_VERSION: ::c_int = 96;
-
+#[cfg(any(target_arch = "x86_64"))]
 pub const __SIZEOF_PTHREAD_RWLOCK_T: usize = 56;
+#[cfg(any(target_arch = "x86",
+          target_arch = "arm",
+          target_arch = "asmjs"))]
```

Comment:
Could these `#[cfg]` blocks be avoided by splitting this into a `musl/$arch.rs`-style configuration like with Linux?